api/query.go

```diff
@@ -12,7 +12,8 @@ import (
 )
 
 type BaseRequest struct {
-	// The blob path to a vds in form: container/subpath
+	// The blob url to a vds in form
+	// https://<account>.blob.core.windows.net/<container>/<blob>",
```

Comment:
ouch. This is how it renders
`https://.blob.core.windows.net//",`

Reply:
Good catch!